data/fleet.csv

```diff
@@ -0,0 +1,4 @@
+key,manufacturer,model,price,seats,reach,cruise,takeoff,hourly
```

Comment:
Why does this use `key` compare to the other 2 data which uses `id`?

Reply:
fixed